devlib/module/sched.py

```diff
@@ -268,6 +268,58 @@ def set_kernel_attribute(self, attr, value, verify=True):
         path = '/proc/sys/kernel/sched_' + attr
         self.target.write_value(path, value, verify)
 
+    @property
+    @memoized
+    def has_debug(self):
+        if self.target.config.get('SCHED_DEBUG') != 'y':
+            logger.warn("target's kernel compiled with !CONFIG_SCHED_DEBUG")
+        return self.target.file_exists('/sys/kernel/debug/sched_features')
+
+    def get_features(self):
+        """
+        Get the status of each sched feature
+
+        :returns: a dictionary of features and their "is enabled" status
+        """
+        if not self.has_debug:
+            raise RuntimeError("sched_features not available")
+        feats = self.target.read_value('/sys/kernel/debug/sched_features')
+        features = {}
+        for feat in feats.split():
+            value = True
+            if feat.startswith('NO'):
+                feat = feat.replace('NO_', '')
```

Comment:
Change to `feat.replace('NO_', '', 1)` to guard against the case where a feature has `NO_` somewhere in its name.

Reply:
Good point!